lib/prometheus/client/push.rb

```diff
@@ -14,8 +14,8 @@ module Client
     # Pushgateway.
     class Push
       DEFAULT_GATEWAY = 'http://localhost:9091'.freeze
-      PATH            = '/metrics/jobs/%s'.freeze
-      INSTANCE_PATH   = '/metrics/jobs/%s/instances/%s'.freeze
+      PATH            = '/metrics/job/%s'.freeze
+      INSTANCE_PATH   = '/metrics/job/%s/instances/%s'.freeze
```

Comment:
s/instances/instance/

Reply:
@beorn7 I can change that but I do seem to find the anything mentioned in the README.md regarding the `instances` thing?